CHANGELOG.md

```diff
@@ -1,11 +1,8 @@
-### Changelog 
+# Changelog
 
- All notable changes to this project will be documented in this file. Dates are displayed in UTC.
+#### [v5.12.0](https://github.com/material-extensions/vscode-material-icon-theme/compare/v5.11.1...v5.12.0)
 
- 
-#### [v5.12.0](https://github.com/material-extensions/vscode-material-icon-theme/compare/v5.11.1...v5.12.0) 
-
-> October 23, 2024 
+> October 23, 2024
```

Comment:
This is semantically incorrect. A h1 should be followed with an h2, not an h4.

Also, the file header about notable changes got deleted.
I (for one) also prefer the keepachangelog format for the date, i.e. `## [v5.12.0] - 2024-10-23`, but that's up to @PKief.
Putting the links in the headers is noisy, if you're scripting it, could you also move them down?
Lastly, what if you linked the release page, not the diff?

Reply:
I'll replace h**4** with h**3**, it's not up for debate.

Reply:
Is h3 a typo? [You can't skip header levels.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#navigation) (accessibility reasons)

Reply:
Can we keep this unresolved? h3 still isn't semantic, and I made some other comments as well.